tarantool/response.py

```diff
@@ -248,3 +251,34 @@ def __str__(self):
         return ''.join(output)
 
     __repr__ = __str__
+
+
+class ResponseExecute(Response):
+    @property
+    def lastrowid(self):
```

Comment:
How about `last_autoincrement_id` name (or even just implement `autoincrement_ids`)? There is no such thing as 'rowid' in tarantool, so it looks a bit strange here.

Reply:
Yes, it will be better to correspond to Tarantool api naming it this layer of the connector.  
Changed the naming and the returning value in response. I handle the lastrowid in dbapi this way:  
```
if response.autoincrement_ids:
    self._lastrowid = response.autoincrement_ids[-1]
else:
    self._lastrowid = -1
```